app/Repositories/CacheRepository.php

```diff
@@ -26,16 +34,21 @@ public function retrieve($key)
      */
     public function retrieveMany(array $keys)
     {
+        $idsWithPrefix = [];
+
+        foreach ($keys as $key => $id) {
+            array_push($idsWithPrefix, $this->setPrefix($id));
```

Comment:
Actually can you also double-check this is actually replacing each ID in the array? Either way, might read a little more nicely as a `array_map`... 🗺